### DISCUSSION TITLE
Route Audio to Roon

### DISCUSSION THREAD

**Discussion by ASenna04** [ExistentialAudio/BlackHole#688] [General]
Hi,
This is my first post here.

I am trying to implement Blackhole to route Audio to Roon. Specifically I am trying to send the Apple Music audio stream to Room. I understand this is possible to BlackHole. Please advise or pint me in the right direction.
Thanks,
ASenna04

**Comment by devinroth** [comment]
What have you tried?

**Reply by ASenna04** [reply]
Hi devinroth.

Thanks for responding.

Not much. I was informed I can do this with HQPlayer and BlackHole but have no idea how to do it. I have installed both of the apps and can see them in Roon in Audio settings as endpoints. I can also see Black Hole in Mac Audio Midi Setup. Not sure what I need to do.

Thanks,

**Comment by devinroth** [comment]
Can’t give you any specifics since I don’t use Roon.

Take a look at these guides and see if they help. Or ask the person who gave you the tip.

https://existential.audio/blackhole/support/
